keepalived-vip/main.go

```diff
@@ -85,8 +84,6 @@ func main() {
 		glog.Fatalf("Terminating execution: %v", err)
 	}
 
-	proc.StartReaper()
```

Comment:
@bprashanth no because we send the `SIGHUP` signal and keepalived reloads the configuration file.